node/client/src/lib.rs

```diff
@@ -483,13 +482,10 @@ pub trait RuntimeApiCollection:
 	+ sp_offchain::OffchainWorkerApi<Block>
 	+ sp_session::SessionKeys<Block>
 	+ sp_authority_discovery::AuthorityDiscoveryApi<Block>
-where
-	<Self as sp_api::ApiExt<Block>>::StateBackend: sp_api::StateBackend<BlakeTwo256>,
```

Comment:
This change is introduced by substrate [PR #14612](https://github.com/paritytech/substrate/pull/14612)